fireloop/index.d.ts

```diff
@@ -360,10 +362,3 @@ interface RemoteMethodArgument {
     type: 'any' | 'Array' | 'boolean' | 'Buffer' | 'Date' | 'GeoPoint' | 'null' | 'number' | 'any' | 'string';
     default: string;
 }
-
-declare var ModelConstructor: ModelConstructor;
```

Comment:
You didn't have to remove the ability to import it. The lint rule just expects you to remove the `declare module` wrapper, making it just:
```
declare var fireloop: ModelConstructor;
export = fireloop;
```

Changed the name to `fireloop` so it gets merged with the `fireloop` namespace. Then you can `import { Filter } from "fireloop";` if `Filter` is moved inside the `fireloop` namespace.